File: 6-Patron/Java/Bota.java
package patron6;

public class Bota extends Botas{
    private int velocidad;

    public Bota(int velocidad) {
        this.velocidad = velocidad;
    }

    public int getVelocidad() {
        return velocidad;
    }

    public void setVelocidad(int velocidad) {
        this.velocidad = velocidad;
    }
    
    
    @Override
    public void verBotas() {
        System.out.println("Velocidad: "+ this.velocidad);
    }
    
}

File: 6-Patron/php/Bota.php
<?php
include './Botas.php';
class Bota extends Botas{
	private $velocidad;

	public function __construct($velocidad){
    	$this->velocidad = $velocidad;
	}

	public function getVelocidad(){
		return $this->velocidad;	
	}

	public function setVelocidad($velocidad){
		$this->velocidad = $velocidad;
	}



	public function verBotas(){
		echo 'Velocidad: ' . $this->velocidad;
	}
}
?>
